CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private IGetJoke _JokeService;
+        private IPrint _PrintService;
+
+        public Jester(IGetJoke jokeService, IPrint printService)
+        {
+            _JokeService = jokeService ?? throw new ArgumentNullException(nameof(jokeService));
+            _PrintService = printService ?? throw new ArgumentNullException(nameof(printService));
+        }
+
+        public void TellJoke()
+        {
+            string joke = _JokeService.GetJoke();
+
+            if (joke == null) { throw new ArgumentNullException(nameof(joke)); }
```

Comment:
```suggestion
            if (joke == null) { throw new ArgumentNullException(nameof(joke)); }
```
You shouldn't need this here